### PR TITLE
Add missing definitions in monorail 2.0 yaml file

### PR DESCRIPTION
**Pull request by keedya**
@RackHD/corecommitters @brianparry @srinia6 @BillyAbildgaard @dalebremner @geoff-reid

### CODE REVIEW

File: spec/lib/api/2.0/lookups-spec.js [hunk 1]
```diff
@@ -121,7 +121,6 @@ describe('Http.Api.Lookup 2.0', function () {
                     return helper.request().post('/api/2.0/lookups')
                         .send(data)
                         .expect(400)
-                        .expect(/ValidationError: /);
```
Comment (anhou) [inline spec/lib/api/2.0/lookups-spec.js]: please keep file mode as 644, don't change to 755

**Comment by anhou** [comment]
:+1:

**Comment by keedya** [comment]
@anhou thanks for your review. I updated the files mode

**Comment by brianparry** [comment]
Looks good @keedya :+1:

**Pull request merged**